.github/workflows/deploy.yaml

```diff
@@ -196,7 +196,16 @@ jobs:
         echo "This PR should only modify non-production files, and it should be auto-approved shortly." >> /tmp/body.txt
         echo "Once it's auto-approved, it should auto-merge." >> /tmp/body.txt
         gh pr create --assignee @me --fill --body-file /tmp/body.txt
-        gh pr checks --watch --required
+        # Avoid a race condition with checks not existing immediately upon PR creation.
+        TRY=1
```

Comment:
Why are we doing it using a while loop with manual incrementing instead of a for loop with escaping like we did in the other PR? I'd just like to have some consistency when it comes to ways we do certain patterns in our bash scripts.